File: Seccion 12 - Punteros en C++/Seccion_12_Python/04_ejercicio_04.py

def main():
    
    numeros: list = []
    nElementos: int
    menor: int = 99999
    
    nElementos = int(input('Digite el número de elementos: '))
    
    for i in range(nElementos):
        numeros.append(int(input('Digite un número[{}]: '.format(i))))
    
    for val in numeros:
        if val < menor:
            menor = val
    
    print('El menor elemento de numeros es: {}'.format(menor))
    
if __name__ == '__main__':
    main()

File: Seccion 12 - Punteros en C++/Seccion_12_Dart/04_ejercicio_04.dart

import 'dart:io';

void main() {

  int nElementos, menor = 99999;
  List<int> numeros = new List();

  stdout.write('Digite el número de elementos: ');
  nElementos = int.parse(stdin.readLineSync());

  for (int i = 0; i < nElementos; i++) {
    stdout.write('digite un número[$i]: ');
    numeros.add(int.parse(stdin.readLineSync()));
  }

  for (int val in numeros) {
    if (val < menor) {
      menor = val;
    }
  }

  print('El menor elemento de números es: $menor');

}